lib/puppeteer_environment.js

```diff
@@ -0,0 +1,33 @@
+const chalk = require('chalk')
+const NodeEnvironment = require('jest-environment-node')
+const puppeteer = require('puppeteer')
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const DIR = path.join(os.tmpdir(), 'jest_puppeteer_global_setup')
+
+class PuppeteerEnvironment extends NodeEnvironment {
```

Comment:
Would be really good to get some comments throughout this file, especially in the setup function.